### ISSUE TITLE
[Bug] Slider text not visible when longer

### ISSUE THREAD

**Issue by fatcatt013** [apache/echarts#18724]
### Version

5.4.2

### Link to Minimal Reproduction

https://echarts.apache.org/examples/en/editor.html?code=PYBwLglsB2AEC8sDeAoWsDGwA2wBOAXLANoBEAxAMyUCcAHACIBCpAugDRqxjA6QhFU6dGDwQA5uICmhWKQCGADwgBnUlwC-ndIoCCylYK7oAJvLDyiZAEwAGAIwAWALS2A7M6el2cu09ce9gCs3r4OLu6eAGyhpH4RgW6x8QGedMnhqfY0Gf6RdrkJztb2hanW1mX5lFUe1o61xSE-cZn5MS0p-UmdbXXpvXl1OYNFlLaNlKWjrnSu02F5c7aVM7bLNWvLzYsRyx27swFVywOH664j53P2E1ueC61LnqvXnptvXvfBJ9G_9j1PmcnntPFcQUcCvcSr8KrCPhCLvVYTtEXNrAc0cVAVjrMCuujwQTnONflMqjR5hTXAiupTbA01vTUXTXJjWe5qesuUS2pTbtT7I9WfZXoj-bS-Z5Ged-SypfZ2QqcSL8QreXlKVDZcVhVK4UzipLNcUZeKmtSMZaVfq1SbrBqIpTSTNblTXbZXGKUm7bMaXL6zT7PbZ5f5fUrwyGbVHZmVfY7PJ6BR6HvHk0HMm6fqnFenPDGA8m7UWwfntRC3TDUwbDlX_UnTeWw6WrTXC428eXE26XXXPeTXfZ3XXh3742PM-Gxy2Hsch3GF7Ye8OU6O0wv7A2hdKJ55Zzu85uO4eS3PsnvrNvh8iF9YDzfIwGbyeb2edw749YR5XvysvzSAGhkBT4vPOdZ_u-f49t-a6_huEHvABXyITmqEnrBMHFN6WbfleAH3gRoGitiBFQcUWF9k8QTzD-fg0Q4Xq5AxY7GixrhTuxwGDFxkZcTGXF2lxGrsXB9G0UKzESThwQSWxEmcRJYaiXxEkCRJQkSSJtEVuJjG1tROnyfpin6cpOmqfpmn6dpjGDosDF_nqskrExPFeoB7mudxDkebYlnQVJrliQ4jkIYZwUyWFW5BbBpmweZwUBWksVll5eF3L5rnVlleFRR5-HpU2rCaNosAAJ76KoRjCLAKggNgEBgAAMhA0BSDVtV1QAFsAADuRAAGbyNgKhSMYsAaKVXBmBYABavAALZWEgGgcFwY1iFIhjIBN0DyItHVyEwUgQAAVm14iwLoACKACS3gTWA5UgEdpAAEbyHgj21bNlgkMEUQ-E4tjA9QwN0EDsD2HQ1jA_5wPZD4UR0D4QSOD4sM-DQoPQ5QGPQ3QqPQzQBPBDQwNBBT0NRNT9RkzQ1OKpQwOONTdC41EbjA1EcPQ44uNOCz0P3j4bhQ0K1NuILDLw0zVPA_UYNBJTgtRGTtOY8LwQq7Abi6wrevE1uEt-vDZPi5jTPWNzsDo_DgsCxDUN2Jz9hy4j7vQ042O26TyNQ_bsB0LbHOY2TAI88bDqK7b6vI47Fu40ExMh2LyfCzQfNuNHmcGwTKM-K7PhTATlBuNT-Mu-Dwe424wuW3jfMlF7eJ00EuNY7AjM88LGK25QQQN8Tbh80TrO43YLuy7AUQG_LdfU1EmfC2H3fz8jutuLbQvY8T8fd33Xc69jDMuzQq_az7sA24HBcG17vNW5vmPN43wTC0EfOa97q98z3dttZm2hrYOu2sgi63qOA3WopdZzxfnrOmoCi4YkVt_TuMCcbw0gesNGTMoiTyFEXeBsBKCQ0Rqnam_s9Y7y3KzPuQRaE7zoDApGs9dZp1njLY249-bCyfiAwWX8eaBw3tDfWRcL4l1QdDIeYs-5cwhsLWgZMU6Y2nm4Eq6ApoaCAA&_source=echarts-doc-preview

### Steps to Reproduce

Everything present in minimal reproduction example.. Basically what I want is to make the text overflow the container.. seemingly it's a canvas so it might not be possible, I am looking for a way to optimize this.. is there an option to move the slider text to the bottom for example? I have been browsing the docs but couldn't find anything.

### Current Behavior

![image](https://github.com/apache/echarts/assets/54073116/32fa72cf-dc8d-4381-af79-037ac196d750)

### Expected Behavior

Basically what I want is to make the text overflow the container.. seemingly it's a canvas so it might not be possible, I am looking for a way to optimize this.. is there an option to move the slider text to the bottom for example? I have been browsing the docs but couldn't find anything.

### Environment

```markdown
- OS: Ubuntu 22.04
- Browser: Brave
- Framework: Angular 13
```

**Comment by helgasoft** [comment]
> Slider text not visible when longer

use [textStyle](https://echarts.apache.org/en/option.html#dataZoom-slider.textStyle) ? [Demo Code](https://echarts.apache.org/examples/en/editor.html?code=PYBwLglsB2AEC8sDeAoWsAeBBDEDOAXMmurACYCGYFRA2gEQBMADAIwAsAtMwOycexmADgLDRIgJxSJ9ADSwmbLr34BWOQpYdufVgDYNi7StY9DW5bqHmlO_jPlHLnFjeN9GrN88aNvdxgBmfxVGdhCPdUcLAINo21CzePcXa2SfB00EvkDmCM5Ar3TuIW4irONS5j9i4W5g2qqoiuUquJaSnRCqtI665kynTtY8xv5yobrWGr7S1gbZ_nCx1mbJufb1_iTF1l6t1kGYutcxz26XGa2gi7DbtePSxk3Hlx3r_dfGI4TS3IvCiEJGUgfVQcxln1gcwHgloS84V1atDPoiBqCRhiJsdgdMMQtJrjIYS1BiEcZce8cfxURT7KDTlCXNjEb4GQTqXdkS5YXTngyqazacpgd9Qf9iiMQZLmNwrjEpcwObZFcSFbKYf5FeSuIrBdpFcL-BqfgbZZiZeMtea1SrzbzdeadcbttaaW7Dm7GUMpedLWz_cqzS5bcHGA6Xfz_frHakvabYxKOlLAZLWNLk-mlVqs6HdVmI6wszHxiUc9wE6WLZmrWn-EH80ty6tm86i6663tm5X2zc61ya-Hy1HByXe0bez3GBmfdPqlq5w3-HPC3O23Ox3OJ3Op8aF7Xk9P5vuOCfVx3D_TJdOA5e-3fzyPZ2994xty5d0mjKoyjOtD-2DlNwAKzIMQO4UNwM1ZIoPJKD9Sg2koNNcDq2_X8i2AjD5TYVCwIwyCMN5VC4IwhCMKQjCUN_b1_xonDVho_DAIHdCWOImjSJYyiWOowDUwqAC5xZRjqiAmC5TBCSxOgwTJOYLidywsS0LolSRKE5dlKPZij0IlSOJUxT3Wko8-JvUY5LEv0rJvBjNPvNib3CABdEgAF9ZBIABPHB8CIVBSFgPAQAAGwgMAABkIGgABTAKSCCvAAAtgAAdyIAAzChQrwWLEtgdyPK89BKGoAAtYBgAAWzoJBYAKsBYowMAAGUwG80L4vqtKIDIMBkqIdhmHkYAADdYoAJ0y0L0oIAByAAjSbYooABreb5EymAwAAdViiAAHNkrABbFuAUKyHmzyCoAelu0KKEW2LQoAMWASbqqoJrJqIeaAB1oEBwGkDGnKAFdYvc-aPJckrgqmiBYsIYggo6kB4oURaKEmuQCrKmhYFoVY9HkDgRtgeZAjJoRScpoRGDJhSycOeQ9CEeRVHYeQGfkCQKfmbn6Y5ymJCF1YJDJ1RJcpvQZbCcWpDJvRqcp9gZaECm9B4ZXGbVgX2FV6ZVHkHg6aLGWeAFiEmZliWybCMnAhNynVAFvRxblnmjdUF2eBd6XTZF-ZzaVJnxbNnm7cYHXYC5pmDYF2n5BYLXWFtln08pjg-djsW2bp-PYCEWPNZ58XTGV4Pvgd2OPbZg2I4p1QRZL03m9ViQ9Z4avO4DoX2ZT5gKcKIXAh4GXAnYOmglVsvYB4VXI8pwI9c8LO33lt2eb1pXZdV55Y-dpeRZ4PWhGD4ah5nm3YD0AO7e3heZZVvm54piQH7Zv3Y44TuRfrrACQB9eauxdvnUWM9gHexZkLGOhcB4Byznoc-L8XagPgVLVWqg9Ze2znPXeMtVBGzDpTYepsfYuzCJQh2Lt77f1NvLchsBfDm18A3HmLtDjWyocITmds9AU3XinehsBAjJ2zq3GWECeC_3mGTQ2KdVByN_kILhrM77oLrtbYOF8FGqxQQnKWet9CFy_pTf2KdoFiOeFgihIjVGq0CBA1Yrcb48DcugIq7kgA&_source=echarts-doc-preview)
